docks/base/Dockerfile

```diff
@@ -0,0 +1,27 @@
+## goclang/base
+## an ubuntu-based docker container with go.
+from ubuntu:trusty
+maintainer go-clang-devs@github.com/go-clang
```

Comment:
I am a little baffled if we have an official mail address for the group? Do you know if GitHub automatically adds one?